api/analytics/analytic_report.py

```diff
@@ -0,0 +1,100 @@
+from flask import jsonify
+import pandas as pd
+
+from helpers.calendar.analytics import RoomAnalytics
+from .write_files import WriteFile
+
+
+class AnalyticsReport():
+    """Get room analytics to export
+       :methods
+           get_dataframe
+           get_all_rooms_summary
+           get_least_used_rooms
+    """
+
+    def get_dataframe(self, rooms_available, time_start, time_end):
+        """ Get all rooms data in a dataframe
+         :params
+            - rooms_available
+            - time_start, time_end
+        """
+        res = []
+        for room in rooms_available:
+            calendar_events = RoomAnalytics.get_all_events_in_a_room(
+                self, room['calendar_id'], time_start, time_end)
+            if not calendar_events:
+                res.append({'RoomName': room['name'],
+                            'minutes': None, 'summary': None, 'attendees': 0})
+            for event in calendar_events:
+                if event.get('attendees'):
+                    event_details = RoomAnalytics.get_event_details(self, event, room['calendar_id'])  # noqa: E501
+                    event_details['attendees'] = len(event.get('attendees'))
+                    res.append(event_details)
+        return pd.DataFrame(res)
+
+    def get_all_rooms_summary(self, query, week_start, week_end):
+        '''
+        Get a summarry data for all rooms in a dataframe
+        '''
+        rooms_available = RoomAnalytics.get_calendar_id_name(
+            self, query)
+        df = AnalyticsReport.get_dataframe(
```

Comment:
Avoid abbreviated variable names, use something like available rooms, it will be easy to n=know it is a dataframe by seeing the get_dataframe method than the 'df'